src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/RcaController.java

```diff
@@ -85,6 +89,9 @@
   // This needs to be volatile as the RcaConfPoller writes it but the Nanny reads it.
   private static volatile boolean rcaEnabled = false;
 
+  // This needs to be volatile as the RcaConfPoller writes it but the Nanny reads it.
+  private static volatile long lastModifiedInMemory = 0;
```

Comment:
This is in milliseconds right? We should add that to the name ?

Reply:
Right, Will update the variable name to reflect this.